templates/cisco_ios_show_switch_detail.textfsm

```diff
@@ -7,6 +7,7 @@ Value STATE (.+?)
 
 Start
   ^Switch/Stack
+  ^Mac persistency wait time
```

Comment:
Just a really small ask, but instead of the static spacing, can you change to `\s+` between the words on the newly added line?

Once that is fixed, I will get this merged

Reply:
Done